configurator/src/main.rs

```diff
@@ -119,8 +119,8 @@ enum BitcoinCoreConfig {
     None,
     #[serde(rename_all = "kebab-case")]
     Internal { user: String, password: String },
-    #[serde(rename_all = "kebab-case")]
-    InternalProxy { user: String, password: String },
+    // #[serde(rename_all = "kebab-case")]
+    // InternalProxy { user: String, password: String },
```

Comment:
i would remove these comments if they are not needed, same with below

Reply:
Removed